.github/workflows/build_wheels_linux.yml

```diff
@@ -24,21 +24,21 @@ jobs:
           - repository: pytorch/audio
             pre-script: packaging/pre_build_script.sh
             post-script: packaging/post_build_script.sh
+            smoke-test-script: test/smoke_test/smoke_test.py
             package-name: torchaudio
     name: ${{ matrix.repository }}
     uses: pytorch/test-infra/.github/workflows/build_wheels_linux.yml@main
     with:
       repository: ${{ matrix.repository }}
-      ref: nightly
+      ref: ""
```

Comment:
What does this do? At a glance, it might as well be omitted.

Reply:
We want to use the main branch from the pytorch/test-infra repo.